hook-docker/main.go

```diff
@@ -59,8 +60,19 @@ func main() {
 		LogOpts: map[string]string{
 			"syslog-address": fmt.Sprintf("udp://%v:514", cfg.syslogHost),
 		},
+		Insecure_registries: cfg.insecure_registries,
 	}
-	if err := d.writeToDisk("/etc/docker/daemon.json"); err != nil {
+
+	path = "/etc/docker/"
+
+	err = os.MkdirAll(path, os.ModeDir)
```

Comment:
Any reason we needed the directory created separately? Did just calling `writeToDisk` not work?